modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -91,52 +94,80 @@ Example of handling these errors:
 </flow>
 ----
 
+The Logger output looks like this (edited for readability):
+
+[source, logs]
+----
+WARN  ...LoggerMessageProcessor:
+org.mule.runtime.core.internal.message.ErrorBuilder$DeserializableErrorImplementation
+{
+  description=My Error
+  detailedDescription=My Error
+  errorType=APP:MYERROR
```

Comment:
TODO: Notice that the error type MULE:COMPOSITE_ROUTING does not appear in the output, just the error type from created Raise Error.

Reply:
@IvanAndresFritzler will check this

The issue is actually the empty On Error Propagate data that returns after calling the Raise Error component in the component.


```

ERROR 2022-10-31 15:03:52,151 [[MuleRuntime].uber.05: [w-scatter-gather-example].
  w-scatter-gather-exampleFlow.CPU_LITE @194953c6] 
  [processor: w-scatter-gather-exampleFlow/processors/2/route/3/processors/0; 
   event: e7a8db20-5967-11ed-9b15-147ddaaf4f97]
  org.mule.runtime.core.internal.exception.OnErrorPropagateHandler: 
  Exception(s) were found for route(s): 

Route 3: 
********************************************************************************
Message               : My Error Example
Element               : (None)
Element DSL           : (None)
Error type            : (None)
FlowStack             : (None)

```